device_plugins/deploy_dsa.md

```diff
@@ -0,0 +1,56 @@
+# Create Intel DSA Device Plugin CR
+
+## Create a CR via web console
+1.	Go to **Operator** -> **Installed Operators**.
+2.	Open **Intel Device Plugins Operator**.
+3.	Navigate to tab **Intel DSA Device Plugin**.
+4.	Click **Create DSADevicePlugin** -> set correct parameters -> Click **Create** 
+5.	Optional: If you want to make any customizations, select YAML view and edit the details. When you are done, click **Create**.
+
+## Verify via web console
+1.	Verify CR by checking the status of **Workloads** -> **DaemonSet** -> **intel-dsa-plugin**.
+2.	Now `DsaDevicePlugin` is created.
+
+## Create CR via CLI
+Apply the CR yaml file:
+```
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/device_plugins/dsa_device_plugin.yaml
+```
+
+## Verify via CLI
+Verify that the device plugin CR is ready: 
+```
+$ oc get DsaDevicePlugin
+```
+Output: 
+```
+NAME		        DESIRED		READY	NODE SELECTOR	                                    AGE
+dsadeviceplugin-sample  3 	        3       {"intel.feature.node.kubernetes.io/dsa":"true"}     98m
+```
+
+# Verify DSA Device Plugin 
+After the plugin is deployed, use below command to verify DSA resources: 
+```
+$ oc describe node srf-2 | grep dsa.intel.com
+  dsa.intel.com/wq-user-dedicated:  0
+  dsa.intel.com/wq-user-shared:     160
+  dsa.intel.com/wq-user-dedicated:  0
+  dsa.intel.com/wq-user-shared:     160
+ ```
+
+## DSA Resource Configuration
+By default the DSA plugin uses [this configuration file](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/demo/dsa.conf).
+The dsa init container comes with a utility called `accel-config` which takes file as input and configures the DSA hardwares based on that.
+The default configuration has creates dedicated WQs for each DSA device so that it's four groups per device where each groups is with 1 WQ linked to 1 engine.
+Users can customise the config and can use the pre-customised config for their specific use case from [here](https://github.com/intel/idxd-config/tree/stable/contrib/configs)
+There's also a possibility for a node specific congfiguration through passing a nodename via NODE_NAME into initcontainer's environment and passing a node specific profile via configMap volume mount.
```

Comment:
`NODE_NAME` is passed automatically by the operator so that is not needed in the sentence:
```suggestion
There's also a possibility for a node specific configuration by passing a node specific profile via configMap volume mount.
```